### DISCUSSION TITLE
Why cannot get email host or port from email connection, only get from airflow conf?

### DISCUSSION THREAD

**Discussion by JoeJoeYuan** [apache/airflow#28913] [General] [answered]
when I configure email for airflow, I found cannot get email host or port from email connection exclude field username and password, only get from airflow conf.
This is very easy to cause confusion, Since it can be configured in Connection, why not get it?
![image](https://user-images.githubusercontent.com/16537696/212273773-08b9b20c-a862-413a-9c76-ecd6c36dcd4b.png)

**Comment by potiuk** [comment, marked as answer]
Why not contributing a change to make it possible?

**Reply by potiuk** [reply]
Generally speaking email configuration has historically eveolved from config only - now it uses user/pass from the connection (an improvement to use only "authentication" from it. The next logical step is to use host -port . And contributions to do that are most welcome.

**Reply by JoeJoeYuan** [reply]
yeah, got it, thanks! maybe I think it is one special design.
I will take a time to make a change!
